README.md

```diff
@@ -23,6 +17,13 @@ Pi Video Machine - a scalable, synchronized, and networked-controlled, raspberry
 
 ## Installation
 ### Prerequisites
+
+### OMXPlayer
+
+To build the customized OMXPlayer, run the command below:
+
+`./build_omxplayer.sh`
```

Comment:
Thank you for adding this. It is definitely a needed subsection in `## Installation`.
However, it is expected that users will be building OMXPlayer *after* they flashed the Raspberry Pi OS and they are able to access the rpi device.

Reply:
Moved this section to after the OS installation